Directory.Packages.props

```diff
@@ -10,8 +10,8 @@
     <GlobalPackageReference Include="Umbraco.GitVersioning.Extensions" Version="0.2.0" />
   </ItemGroup>
   <ItemGroup>
-    <PackageVersion Include="Stripe.net" Version="[41.0.0, 42)" />
-    <PackageVersion Include="Umbraco.Commerce.Core" Version="[12.0.0, 13)" />
-    <PackageVersion Include="Microsoft.AspNet.WebApi.Client" Version="[5.2.7, 6)" />
+    <PackageVersion Include="Stripe.net" Version="43.4.0" />
+    <PackageVersion Include="Umbraco.Commerce.Core" Version="13.0.0-rc1" />
+    <PackageVersion Include="Microsoft.AspNet.WebApi.Client" Version="6.0.0" />
```

Comment:
Please use ranges capped to the next major version rather than explicit versions. Having explicit version number can cause problems if someone needs to use a slightly newer but backwards compatible version but an explicit version number here would block that.